src/ops/mod.rs

```diff
@@ -19,15 +19,7 @@ pub type OpResult = Result<Option<String>, OpError>;
 
 impl From<reqwest::Error> for OpError {
     fn from(error: reqwest::Error) -> Self {
-        let info = if error.is_client_error() {
-            "client error: "
```

Comment:
Why did you remove all of this error messages?

Reply:
Most of the methods were removed from the error type: https://docs.rs/reqwest/0.10.8/reqwest/struct.Error.html